### PR TITLE
Updated rules for somethingawful.com

### PR DESCRIPTION
**Pull request by rufoa**
SA has now deployed SSL on most parts of the site so here are the updated rules. Many thanks!

### CODE REVIEW

File: src/chrome/content/rules/Something-Awful.xml [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
+	<target host="videos.somethingawful.com" />
 
 
 	<securecookie host="^secure\.somethingawful\.com$" name=".+" />
+	<securecookie host="^forums\.somethingawful\.com$" name=".+" />
+	<securecookie host="^archives\.somethingawful\.com$" name=".+" />
+
 
+	<rule from="^http://somethingawful\.com/"
+		to="https://www.somethingawful.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Something-Awful.xml]: This shouldn't be necessary.

File: src/chrome/content/rules/Something-Awful.xml [hunk 2]
```diff
@@ -1,21 +1,29 @@
-<!--
-	Nonfunctional subdomains:
-
-		- (www.)	(shows secure)
-		- fi
-		- forums	(times out)
-		- i
-
--->
 <ruleset name="Something Awful (partial)">
```
Comment (J0WI) [inline src/chrome/content/rules/Something-Awful.xml]: Is this rule still partial? I don't think so.

File: src/chrome/content/rules/Something-Awful.xml (hunk 1 above)
Comment (rufoa) [inline src/chrome/content/rules/Something-Awful.xml]: Unfortunately it is as the bare domain forces a downgrade to http whereas www. works with https

File: src/chrome/content/rules/Something-Awful.xml (hunk 2 above)
Comment (rufoa) [inline src/chrome/content/rules/Something-Awful.xml]: Not any more - please rename

File: src/chrome/content/rules/Something-Awful.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/rules/Something-Awful.xml]: This doesn't matter.
^ will be redirected to https by us, the server redirects to unsecured www then and we will redirect to www over https again.

File: src/chrome/content/rules/Something-Awful.xml (hunk 2 above)
Comment (J0WI) [inline src/chrome/content/rules/Something-Awful.xml]: You have to rename this yourself in your branch.

File: src/chrome/content/rules/Something-Awful.xml (hunk 1 above)
Comment (rufoa) [inline src/chrome/content/rules/Something-Awful.xml]: Ah right, got it

File: src/chrome/content/rules/Something-Awful.xml (hunk 2 above)
Comment (rufoa) [inline src/chrome/content/rules/Something-Awful.xml]: Will do asap (and the other change)

**Comment by rufoa** [comment]
@J0WI rules now fixed. Many thanks

**Pull request merged**